packages/sn-controls-react/src/fieldcontrols/localization.ts

```diff
@@ -37,6 +37,13 @@ export const defaultLocalization = {
   },
   number: {
     noValue: 'No value set',
+    pageCountValues: {
+      '-1': 'No preview provider',
+      '-2': 'Preview generation postponed',
+      '-3': 'Preview error',
+      '-4': 'Preview generation in progress',
+      '-5': 'Preview generation in progress',
```

Comment:
Pls check if these texts and values correspond to the list in the issue.

Reply:
Sorry... I will correct it !